web-client/integration-tests-public/unauthedUserSeesStrickenDocketEntry.test.js

```diff
@@ -86,7 +86,8 @@ describe('Unauthed user views stricken docket entry for order', () => {
   unauthedUserAttemptsToViewStrickenDocumentUnsuccessfully(test);
 });
 
-describe('Unauthed user searches for stricken order', () => {
+// Temporarily disabled for story 7387
+describe.skip('Unauthed user searches for stricken order', () => {
```

Comment:
I just want to check that there's a process set up for unskipping these once the order and opinion searches are reenabled - would that be an AC on an existing issue, or making sure that another issue is logged to unskip these?

Reply:
Comment and link added in AC at https://github.com/flexion/ef-cms/issues/7368